reactDashboard/src/components/GeoHeatMap/GeoHeatMap.css

```diff
@@ -81,3 +98,14 @@ div.leaflet-popup-content-wrapper {
   border-bottom: 1px solid LightGray;
   padding-bottom: 5px;
 }
+
+.leaflet-container .leaflet-control-attribution,
+.leaflet-container .leaflet-control-attribution a {
+  color: black;
+}
+
+.darkmode .leaflet-container .leaflet-control-attribution,
+.darkmode .leaflet-container .leaflet-control-attribution a {
+  background: #0b0a0a;
```

Comment:
background-color ?

Reply:
Surprisingly `background` worked but I have changed it to `background-color` anyway. `background` must be a leaflet thing?

Reply:
background is a css catchall for all background attributes - better to be specific/consistent